packages/renderer/src/lib/image/ImageDetails.svelte

```diff
@@ -64,73 +70,28 @@ function closeModals(): void {
   renameImageModal = false;
 }
 
-let imageInfo: ImageInfo | undefined = $state();
-let image: ImageInfoUI | undefined = $state();
+let imageInfo: ImageInfo | undefined = $derived.by(() => {
+  return allImages?.find(c => c.Id === imageID && c.engineId === engineId);
+});
```

Comment:
You can probably simplify and directly use the store


```suggestion
let imageInfo: ImageInfo | undefined = $derived.by(() => {
  return $imagesInfos.find(c => c.Id === imageID && c.engineId === engineId);
});
```

Reply:
(and remove the allImages derived state)

Reply:
Done